### ISSUE TITLE
iPhone reboots when using Frida

### ISSUE THREAD

**Issue by HGStyle** [frida/frida#2858]
I have a jailbroken iPhone 5S with Sileo, Chimera, and obviously Frida installed on it. On my PC, I can connect manually to my SSH but when using AloneMonkey's frida-ios-dump script, the script connects and quit with the error "Failed to enumerate applications: the connection is closed". As a bonus, my iPhone reboots, meaning that I'll have to re-jailbreak my iPhone since Chimera is "ephemeral" (and it takes my 30 minutes to an hour to get the jailbreak working again).
Please, fix it if you can. For my whole day, I did this. Jailbreak, try the script, but it reboots. Jailbreak, install annother SSH app, try the script, it reboots...

TL;DR: Same issue as https://github.com/frida/frida/issues/2485 and https://github.com/frida/frida/issues/2036

**Comment by HGStyle** [comment]
@oleavr 
![image](https://github.com/frida/frida/assets/84084268/e5087b49-e9d5-412a-b7cc-7074586fede0)

**Comment by Infinitay** [comment]
Have you found a way to fix this?

**Comment by HGStyle** [comment]
@Infinitay unfortunately, no. I'm still waiting for someone to help.
